src/constants.ts

```diff
@@ -26,6 +26,13 @@ export const onlyGitHubSupported: string = localize('onlyGitHubSupported', 'Only
 export const githubAuthProviderId: string = 'github';
 export const githubScopes: string[] = ['repo', 'workflow', 'admin:public_key'];
 
+export const freeSkuDetail: string = localize('freeSkuDetail', 'Free SSL, 2 Custom Domains');
```

Comment:
Typically we only export strings to constants when they are used in multiple places.